tests/backup/backup_test.go

```diff
@@ -4600,6 +4600,193 @@ var _ = Describe("{RestoreEncryptedAndNonEncryptedBackups}", func() {
 
 })
 
+// Share backup with created with same name between two users
+var _ = Describe("{SwapShareBackup}", func() {
+	numberOfUsers := 2
+	users := make([]string, 0)
+	backupNames := make([]string, 0)
+	var contexts []*scheduler.Context
+	var BucketName string
```

Comment:
You dont need this variable here. You can use the variable name as `bucketName` directly in the step implementation where you are creating the backup location